2024_Wendler_et_al_Nonstandard_Numerical_Scheme_IDE/README.md

```diff
@@ -0,0 +1,36 @@
+
+- Section 5.2: In [ide_changepoints](ide_changepoints.cpp), a simulation with both our IDE and an ODE model is performed, where a change in the contact rate is applied after two days to investigate the model behavior at change points. For this scenario, we use epidemiological parameters that are realistic for COVID-19. 
+
+    For the stay time distributions in the respective compartments, we use lognormal distributions as in [Covasim: An agent-based model of COVID-19 dynamics and interventions](https://doi.org/10.1371/journal.pcbi.1009149). There, the mean and the standard deviation are given. In order to use it with our implementation, we compute the corresponding shape and scale parameters with [get_lognormal_parameters](get_lognormal_parameters.py).
+
+    For the transition probabilities we use data from [Assessment of effective mitigation and prediction of the spread of SARS-CoV-2 in Germany using demographic information and spatial resolution](https://doi.org/10.1016/j.mbs.2021.108648) and [Covasim: An agent-based model of COVID-19 dynamics and interventions](https://doi.org/10.1371/journal.pcbi.1009149). There, the data is age-resolved. We consider a model without age resolution which is why we compute according parameters by averaging over the age groups. This is done in [compute_parameters](compute_parameters.cpp).
+
+
+The results can be visualized with [plot_changepoints](plot_changepoints.py).
+
+- Section 5.3: The COVID-19 inspired scenario can be run with [ide_covid_inspired_scenario](ide_covid_inspired_scenario.cpp). We use the same stay time distributions and transition probabilities as in the changepoint scenario of Section 5.2. 
+
+    For initialization of the models we use data reported from RKI that was averaged using a moving_average of 7. The required data set can be downloaded with [download_reported_data](download_reported_data.py). 
+    
+    In the simulation, we set the contact rate such that the daily new transmissions of the IDE simulation are consistent with the reported data at the beginning of the simulation. The contact scaling and the visualization are done in [run_and_plot_covid_inspired_scenario](run_and_plot_covid_inspired_scenario.py). For the scaling and the subsequent visualization we use case data from RKI (not averaged) and data on ICU patients as reported from DIVI. The necessary data can be downloaded with [download_reported_data](download_reported_data.py).
```

Comment:
Maybe rewrite the part explaining the run_and_plot_covid_inspired_scenario file and state that this file automatically runs the ide_covid_inspired_scenario file.